src/shared/lib/MutationUtils.tsx

```diff
@@ -565,3 +571,66 @@ export function createAnnotatedProteinImpactTypeFilter(
     };
     return filter;
 }
+
+export function createNumericalFilter(getData: (d: Mutation) => number | null) {
+    const filter = (filter: NumericalFilter, mutation: Mutation) => {
+        const value = getData(mutation);
+        const lowerBound = filter.values[0];
+        const upperBound = filter.values[1];
```

Comment:
This actually a single value. We should define our `NumericalFilter` value something like this:

```JavaScript
{
    lowerBound: number;
    upperBound: number;
}
```